### DISCUSSION TITLE
Sniff Zigbee 3.0 Traffic with the CC2531 Stick

### DISCUSSION THREAD

**Discussion by clemwo** [Koenkk/zigbee2mqtt#7946] [General]
Hi,
for my thesis in university I'm looking at the Zigbee implementations of various vendors. For this I have a Raspberry Pi 3B+ running Home Assistant, zigbee2mqtt and have a CC2531 Stick connected.  
For sniffing I follow [the guide from zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_sniff_zigbee_traffic.html) and can sniff my Zigbee Traffic originating from Home Assistant just fine.  
Now I would like to compare the "regular" Zigbee Traffic with newer Zigbee 3.0 traffic. Im especially interested in install codes on devices as they promise a higher security since the network key is never transmitted out in the open (or in the de-facto open when it is only encrypted using the standard ZigBeeAlliance09 global Trust Center Link Key)  
Now I have bought another Tuya Zigbee 3.0 Gateway and a LEDVANCE Smart + Plug that has a QR Code printed on it which leads me to believe that this should be an install code. If I connect the Smart Plug to the Gateway with the "Smart Life" App on Android it gets added without the need to scan an install code and I also cant really figure out in which Zigbee Channel the network works in as there is one channel that has A LOT of Zigbee traffic but I cant seem to find neither the MAC Address of the Gateway nor the Smart Plug.  
Now my question is: 
Does the CC2531 stick sniff out Zigbee 3.0 traffic or am I looking for something I cant really find?  
Does anyone have a suggestion what would be good devices to try to sniff out install codes? Im just interested in how different manufacturers implement Zigbee 3.0
I would appreciate some help here and if I find out something interesting I will obviously share my findings with you. Thanks!

**Comment by OsteHovel** [comment]
I had no issues sniffing out Zigbee 3.0 and Zigbee 3 Green Power utilizing the guide you linked to, using ZBOSS and a CC2531
![image](https://user-images.githubusercontent.com/20069958/124721179-2a299500-df09-11eb-951c-7268855112c0.png)

**Reply by clemwo** [reply]
Completely forgot to thank you for your reply, so thank you very much!
